src/app/participant-page/participant-page.component.ts

```diff
@@ -309,8 +313,17 @@ export class ParticipantPageComponent implements OnInit, OnDestroy {
       if (this.participant.data.status === undefined || this.participant.data.status.indexOf( Statics.CONSENT_SUSPENDED ) == -1) {
         this.participantNotConsented = false;
       }
-      if (this.participant.data.dsm != null && this.participant.data.dsm[ "pdfs" ] != null) {
-        this.pdfs = this.participant.data.dsm[ "pdfs" ];
+      this.pdfs = new Array<PDFModel>();
```

Comment:
seems like this part of code from line 356 to line 327 does the same as in `medical-record` extracting it as a method in `PDFModel` class would help to avoid code duplication.